Lib/pathlib.py

```diff
@@ -120,68 +120,18 @@ class _WindowsFlavour(_Flavour):
 
     is_supported = (os.name == 'nt')
 
-    drive_letters = set('abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ')
-    ext_namespace_prefix = '\\\\?\\'
-
     reserved_names = (
         {'CON', 'PRN', 'AUX', 'NUL', 'CONIN$', 'CONOUT$'} |
         {'COM%s' % c for c in '123456789\xb9\xb2\xb3'} |
         {'LPT%s' % c for c in '123456789\xb9\xb2\xb3'}
         )
 
-    # Interesting findings about extended paths:
-    # * '\\?\c:\a' is an extended path, which bypasses normal Windows API
-    #   path processing. Thus relative paths are not resolved and slash is not
-    #   translated to backslash. It has the native NT path limit of 32767
-    #   characters, but a bit less after resolving device symbolic links,
-    #   such as '\??\C:' => '\Device\HarddiskVolume2'.
-    # * '\\?\c:/a' looks for a device named 'C:/a' because slash is a
-    #   regular name character in the object namespace.
-    # * '\\?\c:\foo/bar' is invalid because '/' is illegal in NT filesystems.
-    #   The only path separator at the filesystem level is backslash.
-    # * '//?/c:\a' and '//?/c:/a' are effectively equivalent to '\\.\c:\a' and
-    #   thus limited to MAX_PATH.
```

Comment:
What about these cases? All slashes are replaced with backslashes at the beginning of `splitdrive()`, so invalid paths are now silently converted.

Reply:
Backslashes are already converted to forward slashes in pathlib - check the `parse_parts()` function, which is the only place `splitroot()` is called:

First case:

```python
            if altsep:
                part = part.replace(altsep, sep)
            drv, root, rel = self.splitroot(part)
```

Second case:

```python
                        if altsep:
                            part = part.replace(altsep, sep)
                        drv = self.splitroot(part)[0]
```

As such I think the comment is misleading!

Reply:
I wasn't intending to mislead anyone with these comments. They're from this [patch](https://bugs.python.org/file44588/issue_27827_01.patch). They were intended to clarify the original "interesting findings" with details about the usage and behavior of extended paths in the *Windows API*, not necessarily in pathlib.

Reply:
Ah ha! I misunderstood, sorry. Perhaps I should move this comment into `ntpath.py`?